bundle/internal/tf/schema/resource_registered_model.go

```diff
@@ -0,0 +1,11 @@
+// Generated from Databricks Terraform provider schema. DO NOT EDIT.
```

Comment:
Temporarily added this file while waiting for https://github.com/databricks/terraform-provider-databricks/pull/2771 to merge and the new provider version to be released.

Reply:
We'll take care of a regen + provider version bump once the TF release is done. Then we don't need to piggyback the other changes in the provider in this PR.

Reply:
Sounds good! Removed the temp files